scripts/linux/runE2ETest.sh

```diff
@@ -468,6 +468,11 @@ function process_args() {
     echo 'Required parameters are provided'
 }
 
+function get_random_id()
```

Comment:
Currently you can find that I create another file named testHelper.sh for connectivity case, the intent is to break down tests into its own file.  What does it mean? let's take runE2ETest.sh as an example, we started this script from first tempSensor test.  And ppls add more and more tests to it.  However it is getting mess due to different tests need different input parameters; which means runE2ETest.sh contains the union of all the input parameters for all supported tests.  To solve this issue, we should break down our implementation into its own test files and share some common logic/functions in a shared file (which is testHelper.sh).  Think about even input parameter method (ProcessArgs), we should start sharing by breaking all common parameters in one function as well.

why I explain this?  I think it is good to start with the first step (the easiest one), which is to extract those reusable function in a testHelper.sh file like get_random_id().  Please help to move at least this function to its own file and make sure it is specificed in download artifact step in e2e, LH and stress.

What do you think?